src/main/java/uk/gov/hmcts/reform/sandl/snlrules/controllers/FactMessageController.java

```diff
@@ -21,7 +21,7 @@
     @Autowired
     private FactMessageHandlerFactory factMessageHandlerFactory;
 
-    @RequestMapping(value = "/msg", method = RequestMethod.PUT)
+    @RequestMapping(value = "/msg", method = RequestMethod.POST)
     @ResponseBody
     public ResponseEntity<List<FactModification>> msgw(@RequestBody FactMessage factMessage) {
```

Comment:
was there file too long checkstyle error that the name of this method does not tell me a thing? Either way please find a better one.

Reply:
sure, will merge as is and change in the next PR
this is method that will be receiving events from our events distribution system, there only purpose is to receive message and handle it.